src/components/client/VotingPage.tsx

```diff
@@ -208,7 +208,9 @@ const VotingPage = () => {
       >
         <PageHeader
           header="IQ Voting"
+          hasBody={true}
           body="Follow votes and all related information."
+          hasExternalLink={false}
```

Comment:
You don't need to pass ``hasExternalLink`` and ``hasBody`` have this anymore.
PageHeader type should be ```header: string; body: string | undefined; externalLink: string | undefined```